.github/workflows/lint-test-packages.yml

```
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint.
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Lint and Test Packages
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.12.2"]
```

lets go ahead and add 13 to the matrix!

```suggestion
        python-version: ["3.12.2", "3.13.1"]
```